include/util.h

```diff
@@ -45,4 +45,19 @@ int execve_binary(char *, char *const[], char *const[], char *, int *);
  */
 void format_buffer(char *, int *);
 
+// Create the parent directories needed for a given path iteratively.
```

Comment:
Always use the `/* */` annotation for comments. Especially documentation comments.